.tributors

```diff
@@ -166,7 +166,8 @@
     },
     "AKSoo": {
         "name": "Sin Kim",
-        "blog": "https://orcid.org/0000-0003-4652-3758"
+        "blog": "https://orcid.org/0000-0003-4652-3758",
+        "bio": "I like coding, data, and open science. Current interests are adolescent development and psychiatry."
```

Comment:
may be we should tune up tributors config to not bother about `bio`, not sure if useful